android-project/app/src/main/java/oceantreasur/es/android_project/GameActivity.java

```diff
@@ -147,16 +126,14 @@ public void onClick(View v) {
 
     private void chooseNextActivity(boolean choice, String word) {
```

Comment:
Pass whole CheckAnswerResponse: this way the user of the method won't need to write unnecessary boilerplate code (referencing concrete fields of a concrete object type). ChooseNextActivity can't be called without having a CheckAnswerResponse object, so passing the whole object is logically correct.